ucfcs/GrowthPlus
# Forgot Password Edits and Button click effects

This pr adds translations to the forgot password screen as well as a back button. It also adds button clicks throughout the app and a few styling changes. 

TEST
- Run the app and make sure the forgot password page is translating properly 
- Navigate through the app and make sure each button that is pressed has a pressed effect

Closes #200.
